### PR TITLE
C24_WMDE_Desktop_DE_01

### PR DESCRIPTION
**Pull request by Sperling-0**
- The banners are based on the control banner of desktop-de-25 (2023)
- Following changes are for both the banners:
         - The progress bar is removed
         - The soft-close feature is removed/not used
         - Closing the banner should set a cookie to prevent banner display for 7 days
         - Closing the banner should trigger a banner closed event
         - The already-donated feature is removed/not used
         - The visitors-vs-donors sentence is removed
         - The campaign sentence is not removed
         - The copy is changed
         - The feature to redirect users directly to the payment providers is removed
         - The removed elements are made up for, so the banners share the same height with the base (campaign) banner
         - The larger fallback banner uses a different highlighted sentence
         - The progress bar is removed from the bigger+sliding fallback banner

Ticket: https://phabricator.wikimedia.org/T366159

### CODE REVIEW

File: banners/desktop/C24_WMDE_Desktop_DE_01/banner_ctrl.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	impressionCount,
+	translator,
+	urgencyMessageDaysLeft: 45
+} );
+
+const currencyFormatter = localeFactory.getCurrencyFormatter();
+
+app.provide( 'currencyFormatter', currencyFormatter );
+app.provide( 'formItems', createFormItems( translator, currencyFormatter.euroAmount.bind( currencyFormatter ) ) );
+app.provide( 'formActions', createFormActions( page.getTracking(), impressionCount ) );
```
Comment (Abban) [inline banners/desktop/C24_WMDE_Desktop_DE_01/banner_ctrl.ts]:
```suggestion
app.provide( 'formActions', createFormActions( page.getTracking(), impressionCount, { ap: '0' } ) );
```

File: banners/desktop/C24_WMDE_Desktop_DE_01/banner_var.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	impressionCount,
+	translator,
+	urgencyMessageDaysLeft: 45
+} );
+
+const currencyFormatter = localeFactory.getCurrencyFormatter();
+
+app.provide( 'currencyFormatter', currencyFormatter );
+app.provide( 'formItems', createFormItems( translator, currencyFormatter.euroAmount.bind( currencyFormatter ) ) );
+app.provide( 'formActions', createFormActions( page.getTracking(), impressionCount ) );
```
Comment (Abban) [inline banners/desktop/C24_WMDE_Desktop_DE_01/banner_var.ts]:
```suggestion
app.provide( 'formActions', createFormActions( page.getTracking(), impressionCount, { ap: '1' } ) );
```

**Review by Abban — COMMENTED** [review]
Here's what you need to change to show the correct forms on the application

File: campaign_info.toml [hunk 3]
```diff
@@ -6,24 +6,24 @@
...
+pagename = "B24_WMDE_Desktop_DE_01_ctrl"
+tracking = "org-01-010724-ctrl"
 
 [desktop.banners.var]
-filename = "./banners/desktop/C24_WMDE_Desktop_DE_00/banner_var.ts"
-pagename = "B24_WMDE_Desktop_DE_00_var"
-tracking = "org-00-190624-var"
+filename = "./banners/desktop/C24_WMDE_Desktop_DE_01/banner_var.ts"
+pagename = "B24_WMDE_Desktop_DE_01_var"
+tracking = "org-01-010724-var"
```
Comment (moiikana) [inline campaign_info.toml]: got the info from Kai today that the date format should be yymmdd, I had to change it in the mobile banner too
-> 240701

File: banners/desktop/C24_WMDE_Desktop_DE_01/content/BannerSlides.vue [hunk 4]
```diff
@@ -1,40 +1,37 @@
...
-			{{ campaignDaySentence }} <AnimatedText :content="visitorsVsDonorsSentence"/>
+			Millionen Menschen nutzen Wikipedia, aber 99 % spenden nicht – sie übergehen diesen Aufruf. Die meisten
+			Menschen spenden, weil sie Wikipedia nützlich finden.
 		</p>
 	</KeenSliderSlide>
 	<KeenSliderSlide :is-current="currentSlide === 2">
 		<p>
-			Die meisten Menschen spenden, weil sie Wikipedia nützlich finden. Die durchschnittliche Spende
-			beträgt 22,25&nbsp;€, doch bereits 5&nbsp;€ helfen uns weiter.
+			Die durchschnittliche Spende beträgt xx,xx €, doch bereits 5 € helfen uns weiter. Hat Wikipedia Ihnen in
```
Comment (moiikana) [inline banners/desktop/C24_WMDE_Desktop_DE_01/content/BannerSlides.vue]: Kai updated the numbers in the ticket on Wednesday, need to be updated

**Review by moiikana — APPROVED** [review]

**Pull request merged**